components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/impl/APIMgtAdminServiceImpl.java

```diff
@@ -50,15 +54,46 @@ public APIMgtAdminServiceImpl(APISubscriptionDAO apiSubscriptionDAO, PolicyDAO p
     }
 
     /**
-     * @see org.wso2.carbon.apimgt.core.api.APIMgtAdminService#getAPISubscriptions(int)
+     * @see APIMgtAdminService#getPoliciesByLevel(PolicyLevel)
+     */
+    @Override
```

Comment:
We decided to remove java doc comments from overridden methods, unless there are anything special that you have to mention about this specific implementation.